examples/graphiql-explorer/src/vite-env.d.ts

```diff
@@ -1 +1 @@
-/// <reference types="vite/client" />
+// / <reference types="vite/client" />
```

Comment:
Somehow prettier or our setup of it is flawed and always makes this change, please revert that

Reply:
```suggestion
/// <reference types="vite/client" />
```